ImageFeed/ImagesList/ImagesListViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ViewController.swift
+//  ImageFeed
+//
+//  Created by Артём Костянко on 27.04.23.
+//
+
+import UIKit
+
+class ImagesListViewController: UIViewController {
+    
+    @IBOutlet private var tableView: UITableView!
+    
+    private let photosName: [String] = Array(0..<20).map{ "\($0)" }
```

Comment:
У фигурных скобок всегда должно быть пространство вокруг, особенно у открывающей
```suggestion
    private let photosName: [String] = Array(0..<20).map { "\($0)" }
```